doc/source/_static/js/landing_page.js

```diff
@@ -12,13 +12,17 @@ function collectFamilies(data) {
     if (!project || typeof project !== "object") continue;
 
     const family = project.family;
+    const icon = project.icon || "default-icon.png"; // Fallback icon
```

Comment:
There is no `default-icon.png`. Maybe we could assign the `ansys-icon-light.svg` one? I guess the observer will take care of updating it according to the theme color.

Reply:
good catch, yes we can keep ansys-icon-light

Reply:
added it here [f8450f4](https://github.com/ansys/pyansys/pull/859/commits/f8450f4434f599eb08b931029db6ae9b386dd939)